src/main/java/com/booksaw/betterTeams/metrics/Metrics.java

```diff
@@ -1,226 +1,84 @@
 package com.booksaw.betterTeams.metrics;
 
```

Comment:
Why are we changing this file, it was a direct copy from the bstats website (aside from the added custom metrics) I would prefer not to touch it unless there is good reason to make sure we stay compliant with their TOS

Reply:
I took the latest version from bStats as there was a deprecation warning in there. And it looks you had a version of at least 4y old... So don't worry about the licensing. Only thing I added too was the customChart that you mentioned.